lib/nine_one_one/version.rb

```diff
@@ -1,3 +1,3 @@
 module NineOneOne
-  VERSION = '0.3.0'.freeze
+  VERSION = '0.4.0'.freeze
```

Comment:
I think it's time for version 1 :)
https://semver.org/

Reply:
I think it was a time for 1 long time ago ;) and now it'd be even 2 ;)

Reply:
here you go: "If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.